src/main/java/teammates/logic/core/LogsLogic.java

```diff
@@ -0,0 +1,108 @@
+package teammates.logic.core;
```

Comment:
I don't think there is a need for this file at all - just update the service class with the below methods